frontend/src/libs/components/input/input.tsx

```diff
@@ -5,36 +5,71 @@ import {
   type FieldValues,
 } from 'react-hook-form';
 
+import { getValidClassNames } from '~/libs/helpers/helpers.js';
 import { useFormController } from '~/libs/hooks/hooks.js';
 
+import styles from './styles.module.scss';
+
 type Properties<T extends FieldValues> = {
   control: Control<T, null>;
   errors: FieldErrors<T>;
-  label: string;
+  label?: string;
   name: FieldPath<T>;
   placeholder?: string;
   type?: 'text' | 'email' | 'password';
+  isActive?: boolean;
+  isFilled?: boolean;
+  isDisabled?: boolean;
 };
 
 const Input = <T extends FieldValues>({
   control,
   errors,
-  label,
+  label = '',
   name,
   placeholder = '',
   type = 'text',
+  isActive,
+  isFilled,
+  isDisabled,
 }: Properties<T>): JSX.Element => {
   const { field } = useFormController({ name, control });
 
   const error = errors[name]?.message;
   const hasError = Boolean(error);
+  const inputStyles = [
+    styles.input,
+    isActive && styles.active,
+    isFilled && styles.filled,
```

Comment:
If I get it right - it's for controlling the focus state, not sure we need isFilled prop at all. Just css `:focus` is enough

Reply:
removed isFilled